app/src/main/java/com/example/henzoshimada/feeltrip/ElasticSearchController.java

```diff
@@ -395,13 +401,26 @@ public GetParticipantTask(String username, String password) { // must specify us
 
             ArrayList<Participant> participants = new ArrayList<>();
             String query; // elasticsearch bool queries are amazing in every way
-            query = "{" +
-                    "\"query\" : {" +
-                    "\"bool\" : {" +
-                    "\"must\" : [" +
-                    "{ \"term\": { \"userName\": \"" + username + "\" }}," +
-                    "{ \"term\": { \"password\": \"" + password + "\" }}" +
-                    "]}}}";
+
+            if (searchParticipant){
+                query = "{" +
+                        "\"query\" : {" +
+                        "\"match\" : {" +
+                        "\"userName\" : \"" + username + "\"" +
+                        "}}}";
+            }
```

Comment:
add another case where participant needs to search for a list of users to follow